content/getting-started/usage.md

```diff
@@ -233,11 +234,11 @@ By using Hugo's server in production, you are able to deploy just the source fil
 
 Interested? Here are some great tutorials contributed by Hugo users:
 
-* [hugo, syncthing](http://fredix.xyz/2014/10/hugo-syncthing/) (French) by Frédéric Logier (@fredix)
+* [hugo, syncthing](https://fredix.xyz/2014/10/hugo-syncthing/) (French) by Frédéric Logier (@fredix)
```

Comment:
I wouldn't label `hugo server` as a dev feature. Livereloading is a pretty important feature for some people that just want to preview their content changes immediately. Hugo integrates this feature since a long time and I didn't saw bigger issues with it.